util/src/main/scala/stryker4s/extensions/score/MutationScoreCalculator.scala

```diff
@@ -0,0 +1,13 @@
+package stryker4s.extensions.score
+
+trait MutationScoreCalculator {
+
+  def calculateMutationScore(totalMutants: Double, detectedMutants: Double): Double = {
+    detectedMutants / totalMutants * 100 match {
+      case mutationScore if mutationScore.isNaN =>
+        0.00
```

Comment:
We might also want to log something about having zero mutants. It's a valid use case (dry-run, for example), but not something that should happen without the user being unaware

Reply:
Was thinking about this as well, but i don't think this is the right place for that and maybe not even the correct merge request.